### PR TITLE
Bookstore: Implement styling

### PR DESCRIPTION
**Pull request by CVILLA09**
## Overview
This pull request is part of the "Bookstore: styling" project. The primary objective is to style the Bookstore application to match the design provided in Zeplin.

## Changes Made
- Added missing JSX elements required for styling.
- Applied CSS rules and styles as detailed in Zeplin's design.
- Addressed visual inconsistencies, including layout issues, typography, and color scheme.
- The final design strictly adheres to Zeplin's design, encompassing all elements and styling.

## Testing
The application can be tested by running `npm start` and navigating to `http://localhost:3000`. This allows for adding and removing books with changes being persisted to the server.

## Conclusion
This pull request successfully implements styling as per the Zeplin design, fulfilling all the requirements of the "Bookstore: Styling" project. Your review and feedback are highly appreciated.

### CODE REVIEW

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @CVILLA09 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linter checks failing

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Navigation.js [hunk 1]
```diff
@@ -9,15 +10,17 @@ function Navigation() {
       <nav className="list-container">
         <ul className="list">
           <li className="list-item">
-            <NavLink exact to="/">Home</NavLink>
+            <NavLink className="list-text" exact to="/">Books</NavLink>
           </li>
           <li className="list-item">
-            <NavLink to="/categories">Categories</NavLink>
+            <NavLink className="list-text" to="/categories">Categories</NavLink>
           </li>
```
Comment (Munsa1) [inline src/components/Navigation.js:17]: - [ ] The given font for the navigation is `Montserrat` not `Roboto`, kindly use it asthe point of this project is to follow  the given design 
image
- [ ] Kindly add the user Icon to your navigation as is in the given design

Given design
image

File: src/components/AddBook.js [hunk 2]
```diff
@@ -23,7 +24,7 @@ function AddBook({
   };
 
   return (
-    <button type="button" onClick={handlerAddBook}>
+    <button className="add-btn" type="button" onClick={handlerAddBook}>
       Add Book
     </button>
```
Comment (Munsa1) [inline src/components/AddBook.js:29]: - [ ] Kindly use the correct font  family for the `add` and `update progress` button so as to match the given design. Your design is showing you are using `Arial` while the given design uses `RobotoSlab-Bold` and `light`

Your design
image
image

Given design
image
image

File: src/styles/bookDetails.css [hunk 3]
```diff
@@ -1,13 +1,173 @@
 .book-element--container {
   display: flex;
```
Comment (Munsa1) [inline src/styles/bookDetails.css:3]: - [ ] Kindly use reusable classes for repeating CSS properties for a more DRY codebase. `display: flex;` is repeated 15 times in this file. creating a CSS class then adding the className to affected components is more effiecient. Kindly do so

e.g
```
.d-flex{
display: flex;
}
```

File: src/styles/bookDetails.css [hunk 4]
```diff
@@ -1,13 +1,173 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  height: 100%;
+  width: 33.3%;
+}
+
+.category {
+  display: flex;
+  font-family: "Roboto", sans-serif;
```
Comment (Munsa1) [inline src/styles/bookDetails.css]: - [ ] The same applies for the font-family style. Kindly use a resuable class for it for a more DRY codebase

File: src/components/BookDetails.js [hunk 5]
```diff
@@ -17,27 +22,31 @@ const BookDetails = ({
   return (
     <section className="book-element--container">
       <div className="book-info">
-        <p>{category}</p>
-        <p>{title}</p>
-        <p>{author}</p>
+        <p className="category">{category}</p>
```
Comment (Munsa1) [inline src/components/BookDetails.js]: - [ ] Please use the correct foont for the `category` part of the book details, the correct font is `font-family: Montserrat;` and not `Roboto`

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @CVILLA09, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Inline Styles :heavy_check_mark: 
Good Design :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @CVILLA09   :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- App runs well and functions as expected ✅ 
- Nicely implemented UI styles (Font properties) ✅ 
- Nice attitude to feedback ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**